### PR TITLE
[IMP] sale_timesheet: profitability report

### PR DESCRIPTION
**Pull request by gla-odoo**
In the Project Overview, the Profitability report takes into account
Timesheet costs, but doesn't include information on Revenue.
Because of that, the user is missing crucial information to determine
the profitability of a given project.

This PR aims at fixing this issue by sending the user to the
'Project Costs and Revenues' report of the concerned project when
clicking the 'Profitability' button within the Project Overview page.

It also fixes a tiny labelling typo.

Task-1963932

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_timesheet/views/hr_timesheet_templates.xml [hunk 1]
```diff
@@ -66,7 +66,7 @@
                                             </td>
                                             <td>Timesheets</td>
                                             <th rowspan="6">
-                                                <a type="action" data-model="account.analytic.line" t-att-data-domain="json.dumps(timesheet_domain)" data-context='{"pivot_row_groupby": ["timesheet_invoice_id"], "pivot_column_groupby": ["date:month"],"pivot_measures": ["amount"]}' data-views='[[0, "pivot"], [0, "list"]]' tabindex="-1">Profitability</a>
+                                                <a type="action" data-model="project.profitability.report" t-att-data-domain="json.dumps(timesheet_domain)" data-context="{'group_by_no_leaf':1, 'group_by':[], 'sale_show_order_product_name': 1}" data-views='[[0, "pivot"], [0, "graph"]]' tabindex="-1">Profitability</a>
```
Comment (jem-odoo) [inline addons/sale_timesheet/views/hr_timesheet_templates.xml]: I would rather devine a new domain variable (other than `timesheet_domain`) for project profitability report, even if the domain will be the same. Fields can be different from analytic line and project reporting.
Check in `_plan_prepare_values` to do so ;)

File: addons/sale_timesheet/report/project_profitability_report_analysis.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ class ProfitabilityAnalysis(models.Model):
 
     amount_untaxed_to_invoice = fields.Float("Untaxed Amount To Invoice", digits=(16, 2), readonly=True, group_operator="sum")
     amount_untaxed_invoiced = fields.Float("Untaxed Amount Invoiced", digits=(16, 2), readonly=True, group_operator="sum")
-    expense_amount_untaxed_to_invoice = fields.Float("Untaxed Amount to Re-invoiced", digits=(16, 2), readonly=True, group_operator="sum")
+    expense_amount_untaxed_to_invoice = fields.Float("Untaxed Amount to Re-invoice", digits=(16, 2), readonly=True, group_operator="sum")
```
Comment (jem-odoo) [inline addons/sale_timesheet/report/project_profitability_report_analysis.py]: Should be in a separated commit

**Comment by gla-odoo** [comment]
I pushed the changes, tell me if still problematic!

**Comment by jem-odoo** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**